include/nil/crypto3/zk/commitments/detail/polynomial/proof_of_work.hpp

```diff
@@ -104,19 +104,19 @@ namespace nil {
 
                         while( true ) {
                             transcript_type tmp_transcript = transcript;
-                            tmp_transcript(proof_of_work);
+                            tmp_transcript(std::array<typename FieldType::value_type, 1>{proof_of_work});
```

Comment:
We do this because we want the type-trait to work? Maybe we add a function to transcript that accepts a group element?